oam/install_scripts/columnstore

```diff
@@ -81,7 +81,7 @@ start() {
 		exit 0
 	fi
 
-	($SUDO mkdir -p /var/lock/subsys && $SUDO chmod 777 /var/lock/subsys && $SUDO touch /var/lock/subsys/columnstore) >/dev/null 2>&1
+	($SUDO mkdir -p /var/lock/subsys && $SUDO chmod 755 /var/lock/subsys && $SUDO touch /var/lock/subsys/columnstore) >/dev/null 2>&1
```

Comment:
This appears to chmod to 755 for all users. For root we don't want to do this, for non-root it should be 777